language/tools/read-write-set/src/dynamic_analysis.rs

```diff
@@ -147,8 +152,11 @@ impl ConcretizedFormals {
             } else {
                 AbsAddr::default()
             };
-            new_actuals.push(actual)
+            new_actuals.push(actual);
         }
+
+        let temp_indices = (0..formal_types.len()).collect::<Vec<usize>>();
```

Comment:
I think it would probably make more sense to do this inside `from_args_` (perhaps in the same loop that builds the `sub_map`.